### DISCUSSION TITLE
Add a disable switch for binding to multiple receivers at once.

### DISCUSSION THREAD

**Discussion by Nighthawk-FPV** [ExpressLRS/ExpressLRS#2747] [Ideas]
Hi,

I work at a large company and we have multiple aircraft and transmitters that are all running the same binding phrase. I really like this ability as it allows me to be able to easily swap between aircraft and radios. However, when a coworker tries to fly another aircraft (with another radio) at the same time as me, my radio binds to his aircraft and my aircraft simultaneously which prevents him from binding to his aircraft. 

I would like to request a disable slider in the webui interface that disables an ELRS transmitter from binding to more than one receiver at a time. This would be a huge help. 

Thanks for all the hard work yall put into this amazing system.

Nighthawk

**Comment by pkendall64** [comment]
You are confusing binding with connecting. Binding is a one-time process that means that the devices will be able to connect to each other and follow along the same hopping pattern. Connecting happens when you power up a bound pair and they sync and the receiver listens to the messages from the transmitter. There is no 2 way comms. Think of the connection as 2 independent 1-way comms channels, one for RC and commands, the other for telemetry and command responses.
So, given this you can see that anything with the same bind phrase will connect when powered on. 

For your situation, I would suggest having all the transmitters with their own bind phrase and using Volatile binding on the receiver.
This way after powering up the model, the receiver is in bind mode and you can bind to it from the transmitter then fly. Then when you power-cycle the drone, it will enter bind mode again, then another transmitter can bind to it.

**Comment by Ginkgo-Git** [comment]
貌似这个现象，目前不会发生，我使用两个TX使用相同的UID，第二个接收机即使功率比第一台功率高十倍也不会影响RX的接受，这好像是和ELRS设备的同步机制相关。
